### DISCUSSION TITLE
Quod Libet configuration

### DISCUSSION THREAD

**Discussion by RabanePaco** [quodlibet/quodlibet#4537] [General]
I just started using QuodLibet and I can´t make head or tails of it. I used to use MediaMonkey, the best music mngr on windows an I think Quod Libet can do mostly the same on Linux.
I mostly select my music by choosing genre or composer. How can I make windows to show these like in [https://community.linuxmint.com/software/view/quodlibet] ?

**Comment by declension** [comment]
Try the paned browser, and configure panes as you might want (maybe `genre, composer, album`)

See the [paned browser section in the user guide](https://quodlibet.readthedocs.io/en/latest/guide/browse/paned.html) for more details

**Reply by RabanePaco** [reply]
Thanks
